README.md

```diff
@@ -35,7 +35,7 @@ First, make sure you have the following installed:
 - Next, in an ordinary console, install the `dotnet interactive` global tool:
 
 ```console
-> dotnet tool install -g --add-source "https://dotnet.myget.org/F/dotnet-try/api/v3/index.json" Microsoft.dotnet-interactive
+> dotnet tool install -g --add-source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" Microsoft.dotnet-interactive
```

Comment:
is this the path to use instead of myget?

Reply:
The MyGet feed URL appears in a number of other locations.

Reply:
There should be a separate PR to update the URLs.  The MyGet feed hasn't been updated since January.